server/src/main/java/io/druid/metadata/SQLMetadataConnector.java

```diff
@@ -368,6 +368,22 @@ public void createLockTable(final String tableName, final String entryTypeName)
     );
   }
 
+  public void createOwnerTable(final String tableName, final String entryTypeName) {
```

Comment:
should be, yes. 
can we guarantee rolling release and smooth rollback with altering entry table?